index.html

```diff
@@ -852,11 +852,21 @@ <h2><dfn>CaptureController</dfn></h2>
           </p>
           <pre class="idl">
             [Exposed=Window, SecureContext]
-            interface CaptureController {
+            interface CaptureController : EventTarget {
               constructor();
               undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
+          <div class="issue atrisk">
+            <p>
+              {{CaptureController}} does not yet not define any event handlers.
```

Comment:
```suggestion
              {{CaptureController}} does not yet define any event handler.
```

Reply:
Thanks for catching the superfluous "not" there.

I prefer using the plural "handlers" here because it more naturally conveys that there could be any number of handlers in the future, not a particular one which is currently missing.